packages/react-components/theme-designer/src/components/AccessibilityChecker/AccessibilityChecker.tsx

```diff
@@ -53,9 +53,194 @@ export const AccessibilityChecker: React.FunctionComponent<AccessibilityCheckerP
       'NeutralForeground1 on NeutralBackground1',
     );
     calculateContrastRatio(
-      input.colorNeutralForeground1,
-      input.colorNeutralForeground2,
-      'NeutralBackground1 on NeutralBackground2',
+      input.colorNeutralForeground2BrandHover,
```

Comment:
turn this into data that you loop over. and remove the manual listing of key name. You should be able to pull that from the data.

Reply:
Im new at this been blocked by crooks someone was using my old device to code help would be appreciated been 6 months of lies from everyone please understand its hard to trust anyone